Dockerfile

```diff
@@ -24,21 +24,9 @@ RUN volta run npm ci
 COPY --link . .
 RUN volta run npm run build
 
-
-# build a file server to use for hosting the static files
-FROM rust:alpine AS build-server
-RUN apk upgrade --no-cache
-RUN apk add --no-cache musl-dev build-base
-
-ENV CARGO_HOME=/cargo
-ARG DUFS_VERSION
-RUN cargo install dufs --version $DUFS_VERSION --locked
-
-
 # host the app via the static file server
-FROM alpine:latest AS app 
-RUN apk upgrade --no-cache
+FROM sigoden/dufs
 WORKDIR /wisdom-oss
-COPY --link --from=build-app /app/dist/wisdom-oss/frontend/browser /wisdom-oss/app
-COPY --link --from=build-server /cargo/bin/dufs /wisdom-oss/
-ENTRYPOINT [ "./dufs", "app", "--render-index" ]
+COPY --from=build-app /app/dist/wisdom-oss/frontend/browser /wisdom-oss/app
```

Comment:
Removed link here, since linking this layer always required a rebuild even if no files changed